rush.json

```diff
@@ -621,6 +621,16 @@
       "packageName": "@azure/digital-twins-core",
       "projectFolder": "sdk/digitaltwins/digital-twins-core",
       "versionPolicyName": "client"
+    },
+    {
+      "packageName": "@azure/api-learn",
```

Comment:
Leftover from the tutorial

Reply:
Hey Jeff, you are right in your above comment, I should have made this a draft PR, my bad! Thanks for the comments though. They are very helpful, will work on resolving them and getting this PR in a good state today

Reply:
No worries, friend! Once you're ready, I'll probably have @chradek take a look at it.